main/e2e-testing.md

```diff
@@ -143,3 +143,142 @@ Some examples projects utilizing `@agoric/synpress` for e2e tests:
+
+- `secretWords` _(**string**)_: The secret words for the wallet. (default is a predefined mnemonic phrase that represents a wallet address on the Agoric chain)
+- `privateKey` _(**string**)_: The private key for the wallet.
+- `password` _(**string**)_: The password for the wallet. (default is `Test1234`).
+- `newAccount` _(**boolean**)_: Whether to create a new wallet or not. (default is `false`).
+- `walletName` _(**string**)_: The name of the wallet. (default is `My Wallet`).
+- `selectedChains` _(**array**)_: The chains to select. (default is an empty array, `[]`).
+- `createNewWallet` _(**boolean**)_: Whether to create a new wallet (default is `false`).
+
+**Returns:**
+
+- _(**boolean**)_: `true` if the wallet was set up successfully, `false` otherwise.
+
+## `acceptAccess`
+
+Handles the authorization process for connecting the Keplr wallet to a DApp. Typically, DApps provide a `Connect Wallet` option to initiate this process.
+
+**Returns:**
+
+- _(**boolean**)_: `true` if access was accepted successfully, `false` otherwise.
+
+## `confirmTransaction`
+
+Handles signing transactions within a DApp using the Keplr wallet. This occurs when a user initiates a transaction, such as sending tokens or interacting with a smart contract.
+
+**Returns:**
+
+- _(**boolean**)_: `true` if transaction was approved successfully, `false` otherwise.
+
+## Managing Browser Tabs
+
+In DApp testing, we typically have two important tabs open in the browser: one for the Keplr extension window and another for the Cypress window, where the DApp is running.
+
+- `Keplr Extension Window`: This tab is used for managing your Keplr wallet and interacting with blockchain transactions.
+- `Cypress Window`: This tab runs the DApp you are testing and allows you to interact with the application directly.
+
+`@agoric/synpress` provides the following commands to help determine the status of these windows and switch focus between them:
+
+### `isExtensionWindowActive`
+
+Determines whether the Keplr extension window is the currently active tab.
+
+**Returns:**
+
+- _(**boolean**)_:`true` if the Keplr extension window is active, `false` otherwise.
+
+### `switchToExtensionWindow`
+
+Switches the focus to the Keplr extension window tab.
+
+**Returns:**
+
+- _(**boolean**)_:`true` if the focus change was successful, `false` otherwise.
+
+### `isCypressWindowActive`
+
+Determines whether the Cypress window, where the DApp is running, is the currently active tab.
+
+**Returns:**
+
+- _(**boolean**)_:`true` if the Cypress window is active, `false` otherwise.
+
+### `switchToCypressWindow`
+
+Switches the focus to the Cypress window tab.
+
+**Returns:**
+
+- _(**boolean**)_:`true` if the focus change was successful, `false` otherwise.
+
+## `disconnectWalletFromDapp`
+
+Disconnects the Keplr wallet from all previously connected DApps. After executing this command, the DApps will need to request permission to connect to the wallet again.
```

Comment:
remove "the DApps will need to request permission to connect to the wallet again."